specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-05-01-preview/machineLearningServices.json

```
@@ -1643,6 +1643,423 @@
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Create Blob Datastore": {
+            "$ref": "./examples/createOrUpdateBlobDatastore.json"
+          },
+          "Create File Datastore": {
+            "$ref": "./examples/createOrUpdateFileDatastore.json"
+          },
+          "Create SQL Datastore": {
+            "$ref": "./examples/createOrUpdateSqlDatastore.json"
+          },
+          "Create PostgreSQL Datastore": {
+            "$ref": "./examples/createOrUpdatePostgreSqlDatastore.json"
+          },
+          "Create MySQL Datastore": {
+            "$ref": "./examples/createOrUpdateMySqlDatastore.json"
+          },
+          "Create ADLS  Datastore": {
+            "$ref": "./examples/createOrUpdateAdlsDatastore.json"
+          },
+          "Create ADLS Gen2 Datastore": {
+            "$ref": "./examples/createOrUpdateAdlsGen2Datastore.json"
+          },
+          "Create Databricks File System Datastore": {
+            "$ref": "./examples/createOrUpdateDbfsDatastore.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/DatastoreNameParameter"
+          },
+          {
+            "$ref": "#/parameters/APIVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "properties",
+            "description": "The payload that is used to create or update the Datastore.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/DatastoreCreateRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Datastore creation or update initiated.",
+            "schema": {
+              "$ref": "#/definitions/DatastoreResource"
+            }
+          },
+          "201": {
+            "description": "Datastore creation or update initiated.",
+            "headers": {
+              "Azure-AsyncOperation": {
+                "description": "URI to poll for asynchronous operation status.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the request failed.",
+            "schema": {
+              "$ref": "#/definitions/MachineLearningServiceError"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "MachineLearningDatastores"
+        ],
+        "operationId": "MachineLearningDatastore_Delete",
```

Please change this operationId to align with others
```suggestion
        "operationId": "MachineLearningDatastores_Delete",
```